.github/workflows/fixxxer.yaml

```diff
@@ -8,6 +8,24 @@ permissions:
   contents: write
 
 jobs:
+  reaction:
+    name: Run PR Fixxxer
+    # TODO: support leading and trailing whitespace too
+    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/fixxx' }}
```

Comment:
I understand you used a different job because you want to see a quick reaction and waiting for the `container.image` takes a while?
How do we make sure the `if` conditions do not drift apart? (Perhaps a note to keep in sync will be enough for now as a low-tech solution?)
But I think this should at least do the same check as in the pr-metadata step, otherwise it will give a false sense of progress on PRs raised from other repos.